src/components/desktop/desktop-toolbar.vue

```diff
@@ -6,7 +6,8 @@
   >
     <span
       v-html="$t('totalResults.message', { totalResults: $x.totalResults, query })"
-      class="x-list__item--expand x-font-size--05"
+      class="x-text1 x-list__item--expand"
```

Comment:
Ypu have to change it in mobile too:

![image](https://user-images.githubusercontent.com/6597815/209322411-4bc80f8e-66c7-48d7-bd38-29b2c735f815.png)

It has to be md in mobile

![image](https://user-images.githubusercontent.com/6597815/209322459-e1c2e927-3b89-4646-a8b8-c47730ed5858.png)